src/main/menu.ts

```diff
@@ -194,6 +194,14 @@ function getFileMenu(
       },
       accelerator: 'CmdOrCtrl+N',
     },
+    {
+      label: 'New Test',
+      click: () => {
+        ipcMainManager.send(IpcEvents.CLEAR_CONSOLE);
+        return ipcMainManager.send(IpcEvents.FS_NEW_TEST);
+      },
+      accelerator: 'CmdOrCtrl+T',
```

Comment:
I'm not strongly against this, but does this need an accelerator?

Reply:
I lean toward 'yes' but don't feel strongly about it. Do you want it to be removed?

Reply:
Nah it's alright, like I said I'm not strongly against it. Just was curious because it wasn't something I would have done on my own. 👍